gitpod-io/website
# [Analytics] Track viewport and timezone

## Description


Adds viewport and timezone props in context of all Segment calls.

## Related Issue(s)



Fixes 
[DAT-135](https://linear.app/gitpod/issue/DAT-155/track-simpleanalytics-specific-properties-on-website)

## How to test


1. Open a website Preview
2. Go to [Debugger of Website Staging](https://app.segment.com/gitpod/sources/website_staging/debugger) in Segment
3. Ensure that the `timezone` and `viewport` props are sent in the context of each event

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing


## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.


</details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3893"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
